CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to ESTELA 🕷
+We love your input! We want to make contributing to this project as easy and transparent as possible 🚀, whether it's:
```

Comment:
If we use this emoji here maybe we need to include in all repos and documents, but lets just remove it for now until we have a oficial logo for estela.